charts/core/src/Grid/Grid.tsx

```diff
@@ -26,6 +26,24 @@ export function Grid({ horizontal = true, vertical = true }: GridProps) {
     updatedOptions.xAxis = getUpdatedLineOptions(!!vertical);
     updatedOptions.yAxis = getUpdatedLineOptions(!!horizontal);
     updateChartOptions(updatedOptions);
+
+    return () => {
```

Comment:
This does not get unmounted by default?

Reply:
This issue is that this isn't a real component, in a sense, given that it doesn't actually render anything to the DOM. Instead, this ends up basically being "config via JSX". So on unmount, essentially nothing happens. The component itself, which renders nothing, IS removed from the DOM. But this does nothing to actually update the chart that's rendered, since the chart is config based.